wallet-c/Cargo.toml

```diff
@@ -6,7 +6,10 @@ edition = "2018"
 license = "MIT OR Apache-2.0"
 
 [lib]
-crate-type = [ "staticlib", "cdylib" ]
+crate-type = ["lib"]
+
+[build]
+crate-type = ["staticlib", "cdylib"]
```

Comment:
It just allows me to use the crate when importing it to another crate. But for building just builds the staticlib and cdylib. I had to do this because otherwise, it does not let it compile with the `-lto` flag and let it import from another crate.

Reply:
Moved common code to `/bindings/wallet-core`